decidim-core/lib/decidim/core/engine.rb

```diff
@@ -250,6 +250,12 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "SSL and HSTS" do
+        Rails.application.configure do
+          config.force_ssl = Rails.env.production?
```

Comment:
Should we upgrade the guides for this? Guides, or specify it on the CHANGELOG file, or something.

Reply:
Yeah, the CHANGELOG is still a missing task, when everything's reviewed and done we'll update it.